### DISCUSSION TITLE
Should we define which countries are eligible to become a “country” on Govdirectory?

### DISCUSSION THREAD

**Discussion by RVA2869** [govdirectory/website#495] [General]
### Description

The reason for this question is: The focus country for week 32 is [Bermuda](https://www.wikidata.org/wiki/Q23635). But technically it is not a country at all (or at least not a sovereign country) it is a [British Overseas Territory](https://en.wikipedia.org/wiki/British_Overseas_Territories). 

I realize that the concept of whether or not it is a country is a sensitive subject. There are plenty of examples of countries not recognizing a country.
Hence defining a country.

related issue: #368

**Comment by Ainali** [comment]
That's a good question, and not with an easy, obvious answer. At least the overarching question, about what should be seen on the top level.  There is something about the practicality, and helping the people find their government, that is important here, as it is the purpose for Govdirectory. For example, I think that most people in Taiwan would not expect to find the Taiwanese agencies under China. So I think we should have some sort of *de facto* approach rather than a *de jure* approach. If there is a functioning and organized state with bodies, that should be reflected on Govdirectory.

With that said, for Bermuda, I am still unsure. It is possible that we perhaps just should acknowledge it is an error and that we should put it under the UK, but it is also one of [the participants in the Olympics](https://en.wikipedia.org/wiki/2024_Summer_Olympics#Participating_National_Olympic_Committees), and perhaps they wouldn't expect to go to gov.uk to find info about their agencies.

**Comment by RVA2869** [comment]
We could say only the countries that are sovereign ([https://w.wiki/AuJZ](https://w.wiki/AuJZ)) or [member states of the UN ](https://www.un.org/en/about-us/member-states).

> 
> With that said, for Bermuda, I am still unsure. It is possible that we perhaps just should acknowledge it is an error and that we should put it under the UK, but it is also one of [the participants in the Olympics](https://en.wikipedia.org/wiki/2024_Summer_Olympics#Participating_National_Olympic_Committees), and perhaps they wouldn't expect to go to gov.uk to find info about their agencies.

Well, countries don't participate in the Olympic Games, but National Olympic Committees do. 😉
We will never find a solution that is good for everyone. Or we have to change the content based on geolocation/politics (like Google Maps does). But that is a whole different ball game.

**Comment by Ainali** [comment]
I just again want to make sure we are focusing on the right thing. Our purpose here is to help **citizens** get in contact with who they want to talk to, not to be encyclopedicly correct. 

Perhaps the easiest change is just to change the word "Country" in the menu to "Area"?

**Comment by Abbe98** [comment]
Moved this to a discussion for now. Ultimately I don't think Govdirectory is limited to any type of Government. The European Union is already being worked on, the UN has been discussed before. I personally would consider these more than welcome.
